pkgs/servers/sql/postgresql/pg_config.sh

```diff
@@ -0,0 +1,35 @@
+# The real pg_config needs to be in the same path as the "postgres" binary
```

Comment:
How come there's no shebang here? (Just browsing by as this is blocking my VERY (not) important desktop upgrades where I'm not even using TimeScale yet 😄 )

Reply:
@Ma27 No doubt it'd work. If there's no shebang it'll run in the executing shell. I was just curious why it's omitted. 

I usually use writeScriptBin, stick a shebang in the file as
```#! /usr/bin/env bash```
then prepend a shebang in Nix with a specified version (second shebang is effectively ignored then). The reason being editor tooling reading the shebang to know what highlights and checks to apply.

Sorry I've been up all night and was just curious, call it bikeshedding2.0 in hindsight 😸 

Thanks for the replies!